### ISSUE TITLE
Enhancement - Add Timezone Support

### ISSUE THREAD

**Issue by co-devs** [cyb3rfox/Aurora-Incident-Response#10]
Add timezone columns OR add timezone to existing timezone columns.  Allow users to add items with different timezones and standardize to case configuration default timezone.  Display timezone in visualizations like Visual Timeline.  Add default timezone selection to case configuration (default to UTC).  Very useful for engagements involving investigators and clients spread across multiple timezones.

**Comment by cyb3rfox** [comment]
Good idea. So you are suggesting adding data in different timezones and just normalizing for visualization? That could be done easily. Sorting the grids by timestamps will not work any longer than. Until I can come up with a good solution for that I trust my responders that they are able to convert the tz of the artifact to UTC correctly. I keep that open as I plan a major improvement to the UI that would make what you suggest easier. In that case, every item would have two timestamps where one is with TZ and the other the UTC representation. Sorting happens on the UTC rep.

**Comment by co-devs** [comment]
Yeah I think the best option is, when I go to enter a time stamp, I pick the date, i pick the time, and then i pick the utc offset (or similar).  Whatever offset I put in, the backend adjusts that to UTC+0 and stores it.  UTC+0 should be the default, but I should be able to set that to something else via a drop down in the case configuration (everyone is working from UTC-5 and our client wants everything reported to them in UTC-5, so let's just use that rather than UTC)

But yes I agree, until then any responder should be able to enter times in the correct timezone without the system fixing it for them, and any incident commander should be ensuring that everyone is operating from a standard offset regardless of toolset
